test/parallel/test-trace-events-async-hooks-dynamic.js

```diff
@@ -29,10 +29,10 @@ const proc = cp.spawnSync(
   ['-e', enable + code ],
   {
     cwd: tmpdir.path,
-    env: Object.assign({}, process.env, {
+    env: { ...process.env, ...{
```

Comment:
I think instead of `...{ key: value }` you can use `key: value` directly (here and below)

Reply:
Thank you! Updated.